change/office-ui-fabric-react-2020-04-24-13-39-11-AddingOnDragOverEventForGroupedList.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "Adding onDragOverEventHandler for GroupedList component",
```

Comment:
You can now change it to say adding onDragOver method to IDragDropEvents..

Reply:
Yes that seems nice